lsp/roslyn_ls.lua

```diff
@@ -0,0 +1,186 @@
+---@brief
+---
+-- https://github.com/dotnet/roslyn
```

Comment:
```suggestion
--- https://github.com/dotnet/roslyn
```

for all lines in this comment